.travis.yml

```diff
@@ -0,0 +1,8 @@
+language: python
+python:
+  - "2.7"
+install:
+  - pip install flake8
+cache: pip
+script:
+  - pip install -r test-requirements.txt
```

Comment:
I think this needs to be:
```
install:
  - pip install -r test-requirements.txt
script:
  - flake8
```

As it stands we're only testing whether or not flake8 installs successfully